drivers/hmis_external_apis/lib/tasks/import.rake

```diff
@@ -19,4 +19,133 @@ namespace :import do
+    Hmis::Hud::CustomClientContactPoint.with_deleted.delete_all
+
+    # Delete all CDEs except for Direct Entry flag
+    cdeds = Hmis::Hud::CustomDataElementDefinition.where.not(key: :direct_entry).pluck(:id)
+    Hmis::Hud::CustomDataElement.where(data_element_definition_id: cdeds).delete_all
+
+    # PersonalID => MCI Uniq ID
+    # (Deletes all MCI Unique IDs)
+    HmisExternalApis::ExternalId.where(namespace: :ac_hmis_mci_unique_id).count
+    HmisExternalApis::AcHmis::Migration::InitialMciUniqueIdCreationJob.perform_now(clobber: true)
+
+    # Import MCI Uniq => MCI
+    # (Deletes all MCI IDs)
+    HmisExternalApis::ExternalId.where(namespace: :ac_hmis_mci).count
+    HmisExternalApis::AcHmis::Migration::MciMappingImporterJob.new.best_import_file_key
+    HmisExternalApis::AcHmis::Migration::MciMappingImporterJob.perform_now
+
+    # Pull down CSV
+    today = Date.current.strftime('%Y-%m-%d')
+    dir = "/tmp/migration/#{today}/custom-data"
+    FileUtils.mkdir_p dir
+    creds = GrdaWarehouse::RemoteCredentials::S3.find_by(slug: HmisExternalApis::AcHmis::Importers::S3ZipFilesImporter::MPER_SLUG)
+    s3 = creds.s3
+    zipfile = "#{dir}/custom.zip"
+    s3.list_objects(prefix: 'initial-migration')
+    selected = s3.list_objects(prefix: 'initial-migration').filter { |o| o.key.include?('Custom') }.first.key
+    selected_file_name = selected.gsub('initial-migration/', '')
+    ### ALERT! Check and make sure its the right file. ###
+    s3.fetch(file_name: selected_file_name, prefix: 'initial-migration', target_path: zipfile)
+    system("unzip #{zipfile} -d #{dir}")
+
+    # AC_HMIS_IMPORT_LOG_FILE=/tmp/gig-import.log rails driver:hmis_external_apis:import:ac_custom_data_elements[/tmp/migration/2023-08-30/custom-data,true]
+
+    # Run custom data importers
+    ENV['AC_HMIS_IMPORT_LOG_FILE'] = "/tmp/migration/#{today}-custom-migration-log.txt"
+    importer = HmisExternalApis::AcHmis::Importers::CustomDataElementsImporter.new(dir: dir, clobber: true)
+    importer.run!
+
+    # Upload log file
+    s3.put(file_name: ENV['AC_HMIS_IMPORT_LOG_FILE'], prefix: 'initial-migration')
+
+    # Num referrals by type
+    HmisExternalApis::AcHmis::ReferralPosting.group(:status).count
+
+    HmisExternalApis::AcHmis::Referral.joins(:postings).group(HmisExternalApis::AcHmis::ReferralPosting.arel_table[:status]).count
+    # The below should be 0. All Accepted/Pending should be tied to a household.
+    HmisExternalApis::AcHmis::ReferralPosting.where(status: ['accepted_status', 'accepted_pending_status']).where(HouseholdID: nil).count
+
+    # Count how many MCI and MCI Unique IDs
+    HmisExternalApis::ExternalId.for_clients.group(:namespace).count
+    # Num Clients wihout MCI Unique IDs
+    Hmis::Hud::Client.hmis.left_outer_joins(:ac_hmis_mci_unique_id).where(ac_hmis_mci_unique_id: { id: nil }).count
+    # Num Clients wihout MCI IDs
+    Hmis::Hud::Client.hmis.left_outer_joins(:ac_hmis_mci_ids).where(ac_hmis_mci_ids: { id: nil }).count
```

Comment:
Looks like these counts aren't used. Maybe they should just be comments?

Reply:
Yeah definitely.. I want to clean this up a bit more before merging, it's also just a random function in this file